units/host_monitoring_external_termination.go

```diff
@@ -136,10 +137,19 @@ func handleExternallyTerminatedHost(ctx context.Context, id string, env evergree
 		if cloudStatus != cloud.StatusTerminated && (h.UserHost || h.StartedBy != evergreen.User) {
 			return false, errors.New("non-agent host is not already terminated and should not be terminated")
 		}
-
+		if h.RunningTask != "" {
+			if err := task.AddHostCreateDetails(h.RunningTask, h.Id, ctx.Err()); err != nil {
```

Comment:
It's really likely that there isn't an error in context (so this will noop): I think here instead we're going to want to pass in our own error that tells the user that the host was terminated externally